trunk/SUAVE/Methods/Geometry/Two_Dimensional/Cross_Section/Airfoil/compute_naca_4series.py

```diff
@@ -18,147 +20,67 @@ def compute_naca_4series(camber,camber_loc,thickness,npoints=100,airfoilName='na
     None
 
     Inputs:
-    camber      [-] Maximum camber as a fraction of chord
-    camber_loc  [-] Maximum camber location as a fraction of chord
-    thickness   [-] Maximum thickness as a fraction of chord
-    npoints     [-] Total number of points
+        airfoils              <string>
 
     Outputs:
     airfoil_data.
-        thickness_to_chord 
-        x_coordinates 
-        y_coordinates
-        x_upper_surface
-        x_lower_surface
-        y_upper_surface
-        y_lower_surface
-        camber_coordinates  
+        thickness_to_chord    [unitless]
+        x_coordinates         [meters]
+        y_coordinates         [meters]
+        x_upper_surface       [meters]
+        x_lower_surface       [meters]
+        y_upper_surface       [meters]
+        y_lower_surface       [meters]
+        camber_coordinates    [meters] 
 
     Properties Used:
     N/A
-    """        
-    
-    airfoil_data                    = Data()
-    airfoil_data.x_coordinates      = Data()
-    airfoil_data.y_coordinates      = Data()
-    airfoil_data.thickness_to_chord = Data()
-    airfoil_data.camber_coordinates = Data()
-    airfoil_data.x_upper_surface    = Data()
-    airfoil_data.x_lower_surface    = Data()
-    airfoil_data.y_upper_surface    = Data()
-    airfoil_data.y_lower_surface    = Data()
-    
-    aNames    = [airfoilName]
-    half_pnts = int(npoints/2)
-    
-    upper = []
-    lower = []
+    """         
+    geometry        = Data()
+    half_npoints    = math.floor(npoints/2)      # number of points per side    
```

Comment:
this line is the isssue

Reply:
Good catch, I fixed it so that regardless of even or odd, the panel number does not change.